web/client/epics/search.js

```diff
@@ -321,9 +324,14 @@ export const searchOnStartEpic = (action$, store) =>
                     .switchMap(({ type, geometry, typeName }) => {
                         let coord = pointOnSurface({ type, geometry }).geometry.coordinates;
                         const latlng = {lng: coord[0], lat: coord[1] };
+                        const {x, y} = reproject(coord, 'EPSG:4326', 'EPSG:3857');
 
                         if (coord) { // trigger get feature info
-                            return Rx.Observable.of(featureInfoClick({latlng}, typeName, [typeName], {[typeName]: {cql_filter: cqlFilter}}), showMapinfoMarker());
+                            return Rx.Observable.of(featureInfoClick({latlng}, typeName, [typeName], {[typeName]: {cql_filter: cqlFilter}}), showMapinfoMarker())
```

Comment:
There is a marker on the map after closing the popup and this is due to the showMapinfoMarker() action.
We should avoid to dispatch showMapinfoMarker if the isMapPopup is true bacause the marker should be hidden, the popup is already showing the position.

https://user-images.githubusercontent.com/19175505/194865753-0830262a-05fd-48c1-8a51-a907e1a02234.mp4

Reply:
I also think it's better to hide the marker when we use popup display option.